.github/workflows/acc_test.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           TF_ACC_NETAPP_HOST: ${{ secrets.TF_ACC_NETAPP_HOST }}
           TF_ACC_NETAPP_HOST2: ${{ secrets.TF_ACC_NETAPP_HOST2 }}
           TF_ACC_NETAPP_HOST3: ${{ secrets.TF_ACC_NETAPP_HOST3 }}
+          TF_ACC_NETAPP_HOST4: ${{ secrets.TF_ACC_NETAPP_HOST4 }}
```

Comment:
What host 4?

Reply:
HOST3 not sure what's the IP, so created HOST4 which is 10.193.182.108